common/lib/container-definitions/src/runtime.ts

```diff
@@ -121,7 +122,9 @@ export interface IContainerContext extends IDisposable {
     readonly quorum: IQuorum;
     readonly audience: IAudience | undefined;
     readonly loader: ILoader;
+    // @deprecated
```

Comment:
I think it needs to be:
```suggestion
    /** @deprecated */
```